tests/test_models/test_mixins.py

```diff
@@ -145,55 +149,177 @@ def test_calling_private_prediction(
     )
 
 
-class DummyNN(SaveNNMixin):
-    def __init__(self, a, b):
-        self.a = torch.tensor(a)
-        self.b = torch.tensor(b)
+class DummyDeepBaseModel(SaveDeepBaseModelMixin):
+    def __init__(self, size: int):
+        self.size = size
+        self.net = MLPNet(input_size=size, hidden_size=[size], lr=0.01, loss=torch.nn.MSELoss(), optimizer_params=None)
+        self.trainer = Trainer()
 
 
-def test_save_nn_mixin_save(tmp_path):
-    dummy = DummyNN(a=1, b=2)
+class DummyPytorchForecastingModel(SavePytorchForecastingMixin):
+    def __init__(self, size: int, init_model: bool):
+        self.size = size
+        self.init_model = init_model
+        if init_model:
+            self.model = MLPNet(
```

Comment:
I wasn't very easy to create `DeepAR` model here to be more accurate with the testing.